### ISSUE TITLE
1.12 Lag on Bloomery

### ISSUE THREAD

**Issue by Zygus42** [GlobbyPotato/Rockhounding_OreTiers#7]
rockhounding_oretiers-1.12.2-3.06
rockhounding_core-1.12.2-3.10
forge-1.12.2-14.23.4.2703-installer-win

This is a picture of a lag report using LagGoggles mod.  The Bloomery is one of the highest lagging items in my base currently:
![2018-05-31_15 46 38](https://user-images.githubusercontent.com/20757445/40779448-a84e245c-6507-11e8-92b9-4f00390ed11b.png)

even though it's not actually doing anything:
![2018-05-31_15 46 42](https://user-images.githubusercontent.com/20757445/40779463-b3ef6442-6507-11e8-9863-49a208a65a56.png)

Are you perhaps checking to see if anything is happening too often ?

(Should note that the furnace next to it is only producing lag because it's full and can't do any work - if it was empty or working, would have a lot less lag).

**Comment by GlobbyPotato** [comment]
I think I found the origin of some spikes, will try to make a reduction. On rest it should be between 6 and 15 µs/t. it requires some constant updates for tanks and gui, hopefully won't mess too much.

**Comment by Zygus42** [comment]
Awesome.

**Comment by Zygus42** [comment]
On my latest lag check with lagGoggles, using:
rockhounding_oretiers-1.12.2-3.07
rockhounding_core-1.12.2-3.11
forge-1.12.2-14.23.4.2703-installer-win

The Basic Coal Refiner is my single largest lag machine by a long shot.

This is the Refiner while working (about the same lag as a pen full of 100 chickens):
![2018-06-11_11 25 39](https://user-images.githubusercontent.com/20757445/41211556-c1c662e4-6d6a-11e8-88be-e8bafccf0f80.png)

and while not working (lovely):
![2018-06-11_11 29 22](https://user-images.githubusercontent.com/20757445/41211573-dcbdbf0c-6d6a-11e8-8185-cfdb4c88735c.png)

Considering how slowly it works, it shouldn't be putting so much pressure on the server.

**Comment by GlobbyPotato** [comment]
In my tests it doesn't go higher than 80 while working, considering it checks for heating, recipe and item rendering.
Did you break the machine and place it again after the mod update? I found while testing that it may do crazy lag until refreshed.

**Comment by Zygus42** [comment]
No, I just placed it down recently, well after we patched.

But I picked it up, put it back down, put coal in and retested:
![2018-06-11_17 48 01](https://user-images.githubusercontent.com/20757445/41224730-c052ddea-6d9f-11e8-8dc6-d8de413d2f25.png)

But you are also correct, in a test in singlepayer and server with just your mods and lagGoggles installed, no other mods, the machine is just fine.  Tested with no fire underneath, nether rack fire and lava, as well as the 2 different kinds of coal.

So the conclusion is some kind of cross-mod contamination :(  Hate that.
